lib/railway_ipc/publisher.ex

```diff
@@ -29,6 +29,7 @@ defmodule RailwayIpc.Publisher do
   end
 
   def publish(channel, exchange, message) do
+    message = message |> ensure_uuid()
```

Comment:
I think this is ok here, but long term we should just handle the ensuring of a uuid in the PublishedMessage#create method. This seems like a weird place to ensure the uuid.

Reply:
I'm down to make that change in the future.